src/main/markdown/roadmap.md

```diff
@@ -0,0 +1,18 @@
+Roadmap
+===
+
+We are planing some exciting features and changes for the upcoming releases of the GWT SDK. As usual, when planing something, there can be changes. Things mentioned here are plans and there is no guarantee that the release will contain it.
+
+## Release 2.12.0
+
+*   Deprecating classic `DevMode`
+
+
+*   Catching up on Java language features up to 21 
```

Comment:
https://github.com/gwtproject/gwt/issues/9794 could be added.

Reply:
added

Reply:
Using the comment from @jnehlmeier, this change is no longer relevant.